.yarn/patches/@metamask-assets-controllers-npm-33.0.0-3e7448c4cd.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/dist/chunk-FGAZXVKS.js b/dist/chunk-FGAZXVKS.js
```

Comment:
I don't have a PR to `develop` for this, since I plan to fix that with an actual package release.  But patching for v12.1.0 and probably subsequent minor releases.